Symfony/CS/Fixer/Contrib/ClassKeywordRemoveFixer.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Contrib;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Token;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Sullivan Senechal <soullivaneuh@gmail.com>
+ */
+final class ClassKeywordRemoveFixer extends AbstractFixer
+{
+    /**
+     * @var string[]
+     */
+    private $imports = array();
+
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+
+        $this->storeImports($tokens);
+        $tokens->rewind();
+        $this->replaceClassKeywords($tokens);
+
+        return $tokens->generateCode();
+    }
+
+    /**
+     * @param Tokens $tokens
+     */
+    private function storeImports(Tokens $tokens)
+    {
+        foreach ($tokens->getImportUseIndexes() as $index) {
+            $import = '';
+            while (($index = $tokens->getNextMeaningfulToken($index))) {
+                if ($tokens[$index]->equals(';')) {
+                    break;
+                }
+                $import .= $tokens[$index]->getContent();
```

Comment:
You mean this?

```php
if ($tokens[$index]->equals(';')) {
    break;
}

$import .= $tokens[$index]->getContent();
```

Is that a CS requirement? This was not done by php-cs-fixer.